exercises/static/exercises/global_navigation_newmanager/python_template/ros2_humble/map.py

```diff
@@ -39,25 +41,19 @@ def RTGridWorld(self):
 		return RTz*RTx
 
 	def worldToGrid(self, worldX, worldY):
-		# self.gWidth/self.wWidth is the scale
-		worldX = worldX * self.gridWidth/self.worldWidth
-		worldY = worldY * self.gridHeight/self.worldHeight
-		orig_poses = np.matrix([[worldX], [worldY], [0], [1]])
-		final_poses = self.RTWorldGrid() * orig_poses
-		gridX = int(final_poses.flat[0])
-		gridY = int(final_poses.flat[1])
-		return (gridX, gridY)
+		""" Transform a world point to map cell coordinates """
+		gridX = (worldY + self.worldHeight/2) * self.gridHeight / self.worldHeight
+		gridY = (worldX + self.worldWidth/2) * self.gridWidth / self.worldWidth
+		gridX = int(np.clip(gridX, 0, self.gridWidth))
+		gridY = int(np.clip(gridY, 0, self.gridHeight))
+		return [gridX, gridY]
```

Comment:
Return tuple not array

Reply:
For the GUI it is necessary that both of this functions return a tuple. You can change the deprecated rowColumn() function in the GUI to return a list instead of the tuple

Reply:
Sorry for the miss, I just saw the problem inside the `update_gui()` method when building the message payload.
Fixing and pushing ASAP.

Will still keep the old rowColumn returning a list instead of a tuple just in case.